pkgs/applications/editors/emacs/generic.nix

```diff
@@ -33,6 +33,7 @@
 , srcRepo ? true, autoreconfHook ? null, texinfo ? null
 , siteStart ? ./site-start.el
 , nativeComp ? true
+, version ? version
```

Comment:
What is your idea here?

Reply:
The code sample in the PR description is not possible without this, because .override only acts on the inner function, not the outer (As far as I understand overrides, which is "barely").